### ISSUE TITLE
duplicate icons

### ISSUE THREAD

**Issue by MarceloMJM** [vitalidze/traccar-web#696]
Hello always devenvolvendo an excellent job.
I'm facing this problem, registered as an administrator or manager icons appear duplicate web interface.
The help is welcome .
![print](https://cloud.githubusercontent.com/assets/9204676/15538122/962c3350-2250-11e6-8797-347319cbc6e3.png)
![print1](https://cloud.githubusercontent.com/assets/9204676/15538126/9fe1b474-2250-11e6-86ec-53fd833d1f5b.png)
![print3](https://cloud.githubusercontent.com/assets/9204676/15538129/a544c94c-2250-11e6-86e6-7b88fe05c025.png)

**Comment by vitalidze** [comment]
I don't understand the issue here. You have two devices with same name?

**Comment by MarceloMJM** [comment]
I have multiple devices , but only two of them are showing up in duplicate

**Comment by integreated** [comment]
Hi
Great job.
I have the same problem with duplicate icons.

**Comment by MarceloMJM** [comment]
a detail in the Chrome browser appears the problem, but in Internet Explorer works perfect .

**Comment by integreated** [comment]
I have the problem using Firefox I will try it using Explorer.

**Comment by integreated** [comment]
Tested in Explorer and all worked ok.
I changed the traccar config from 127.0.0.1 to localhost and it now seems ok with Firefox as well.

**Comment by vitalidze** [comment]
@integreated I don't think it is related with the hostname/address. There should be some bug in the code, but I cannot reproduce it myself. Even on server of @MarceloMJM it shows me correct result in google chrome.

**Comment by integreated** [comment]
Hi Vitalidze
Got the problem back again so your right it is not the address.
I have noticed that when it happens I suddenly have the same vehicle showing as a duplicate in the list and sometimes there are several duplicates.
All seems ok in explorer but not everybody uses it.

**Comment by MarceloMJM** [comment]
I updated the browser chrome and apparently solved the problem.
Thank you for Vitalidze support.

**Comment by integreated** [comment]
I have updated Firefox and installed the latest Google and on first login it looks ok but as soon as I zoom in or out I get the duplicate again.

**Comment by vitalidze** [comment]
Well, I need some way to reproduce this issue to figure out why it's happening.

**Comment by integreated** [comment]
Where can I send you the url?

**Comment by vitalidze** [comment]
You can find email in my github profile.

среда, 1 июня 2016 г. пользователь integreated написал:

> Where can I send you the url?
> 
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> https://github.com/vitalidze/traccar-web/issues/696#issuecomment-222819602,
> or mute the thread
> https://github.com/notifications/unsubscribe/ACMCNQESgTDd42OC64bdaM-4SHpKb1gdks5qHKH4gaJpZM4ImbPX
> .

## 

Kind regards, Vitaly Litvak

**Comment by vitalidze** [comment]
Please try latest version and let me know whether issue is still reproduced there.

**Comment by MarceloMJM** [comment]
Unfortunately , there is still the problem with the new version using google chrome .
Will provide a new access to the server

**Comment by integreated** [comment]
Just tested this on Firefox and Google and I don't seem to have the problem anymore.

**Comment by ghost** [comment]
Hi,
Has somebody solved the issue of duplicated devices on the map?
